assets/css/index.css

```diff
@@ -1,128 +1,244 @@
 * {
   box-sizing: border-box;
-  padding: 0;
   margin: 0;
 }
 
 body {
-  font-family: sans-serif;
-  font-size: 16px;
+  position: relative;
+  background: #f4f5f7;
 }
 
-header {
-  position: absolute;
-  width: 375px;
-  height: 44px;
-  left: 1px;
-  top: 1px;
+svg {
+  margin-right: 10px;
 }
 
-nav {
-  padding: 0 15px;
-  margin-top: 5px;
+.navbar {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 100%;
+  height: 10vh;
+}
+
+.logo-placeholder {
+  padding: 10px 12px;
 }
 
-a {
+.logo {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: rgb(96, 112, 255);
+  margin-left: 10px;
   text-decoration: none;
 }
 
-#menu {
+.normal-button {
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  border-radius: 8px;
+  border: none;
+  background-color: #f4f5f7;
+}
+
+.normal-button i {
   color: rgb(96, 112, 255);
   font-size: 18px;
   line-height: 20px;
   margin-right: 15px;
 }
 
-ul {
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  height: 90vh;
+  background-image: url("../img/index-bg.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.primary-txt {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 50px;
+  color: #172b4d;
+  margin-bottom: 10px;
+}
+
+.connect {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  text-decoration: none;
+}
+
+.connect-icons ul {
   list-style-type: none;
+  padding: 0;
+  padding-top: 10px;
 }
 
-li {
+.frame-client-detail li {
   display: inline;
+  padding-left: 20px;
 }
 
-svg {
-  margin-right: 10px;
+.connect-icons ul li {
+  display: inline;
 }
 
-.navbar {
+.works {
+  position: absolute;
+}
```

Comment:
- [ ] it is required for this project. to complete this section using a grid layout. Kindly change the styles to achieve the same design using grid layout.